src/app/data/calculations.ts

```diff
@@ -1,5 +1,6 @@
 import { Metric, ScoutMetricType } from '@robot-analytics/data/metric';
-import { reduce, map } from 'lodash';
+import { reduce } from 'lodash';
+import { std, round } from 'mathjs'
```

Comment:
Missing semicolon (semicolon rules aren't strictly enforced -- we should get linting set up soon!)

```suggestion
import { std, round } from 'mathjs';
```